workspaces/liferay-sample-workspace/client-extensions/liferay-sample-custom-element-2/src/index.js

```diff
@@ -37,19 +40,29 @@ class WebComponent extends HTMLElement {
 		super();
 
 		try {
-			this.oAuth2Client = Liferay.OAuth2Client.FromUserAgentApplication(
-				'liferay-sample-oauth-application-user-agent'
+			this.oAuth2Client1 = Liferay.OAuth2Client.FromUserAgentApplication(
```

Comment:
I wonder if these objects might not be a little more expressively named `oAuth2ClientSpringboot` and `oAuth2ClientNode` just so you have an idea as you're reading through what these actually are.